apps/onboarding/src/app/result/components/ResultCard.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled';
+import { text } from '@sulsul/token/src/text';
+import { getLevelDetails } from '../service';
+import { colors } from '@sulsul/token/src/colors';
+import { useSearchParams } from 'next/navigation';
+
+const Heading1 = styled.h1`
+  ${text.heading[1]};
+  color: white;
+  letter-spacing: -0.3px;
+  margin-bottom: 4px;
+`;
+
+const BorderWrapper = styled.div`
+  margin: auto;
+  padding: 1px;
+  width: 100%;
+  max-width: 328px;
+  border-radius: 16px;
+  background-image: linear-gradient(#23272e, #23272e),
+    linear-gradient(135deg, #23272e, #fff8, #23272e);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+  overflow: hidden;
+`;
+
+const ResultWrapper = styled.div<{ mainColor: string }>`
+  position: relative;
+  padding: 20px 32px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  background: url('/icons/grainy.svg') center center / contain;
+
+  &::after {
+    content: '';
+    width: 10px;
+    height: 10px;
+    position: absolute;
+    bottom: -20px;
+    left: 50%;
+    transform: translateX(-50%);
+    border-radius: 50%;
+    background-color: ${({ mainColor }) => mainColor};
+    box-shadow: 0px 0px 100px 100px ${({ mainColor }) => mainColor};
+  }
+`;
+
+const Badge = styled.span`
+  border-radius: 8px;
+  padding: 4px 8px;
+  margin-bottom: 16px;
+  color: ${colors.purple};
+  background: rgba(187, 182, 255, 0.16);
+`;
+
+const SVGWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  & svg {
+    width: auto;
+    height: auto;
+  }
+`;
+
+export const ResultCard = () => {
+  const searchParams = useSearchParams();
+  const result = Number(searchParams.get('result'));
+  const { name, svg, description, mainColor } = getLevelDetails(result);
+  return (
+    <BorderWrapper>
+      <ResultWrapper mainColor={mainColor}>
+        <Heading1>{name}</Heading1>
+        <Badge>{description}</Badge>
+        <SVGWrapper>{svg}</SVGWrapper>
+      </ResultWrapper>
+    </BorderWrapper>
+  );
+};
```

Comment:
ResultCard를 여러 페이지에서 재사용하려면 useSearchParams에 대한 의존도를 낮추고 필요한 props를 외부에서 제공하는 방식으로 변경하면 더 좋을 거 같아

```suggestion
export const ResultCard = ({ name, svg, description, mainColor }) => {
  return (
    <BorderWrapper>
      <ResultWrapper mainColor={mainColor}>
        <Heading1>{name}</Heading1>
        <Badge>{description}</Badge>
        <SVGWrapper>{svg}</SVGWrapper>
      </ResultWrapper>
    </BorderWrapper>
  );
};
```

Reply:
ResultCard가 다른 곳에서 또 쓰일 일이 있나..? 없다고 판단해서 Result 폴더에 component로 넣어뒀어.

> useSearchParams에 대한 의존도를 낮추고 필요한 props를 외부에서 제공하는 방식으로 변경하면 더 좋을 거 같아
이건 동의! 수정해둘게